numpy/_core/tests/test_scalarmath.py

```diff
@@ -918,6 +922,15 @@ def test_longdouble_operators_with_obj(sctype, op):
         pass
 
 
+@pytest.mark.parametrize("op", [operator.add, operator.pow, operator.sub])
+@pytest.mark.parametrize("sctype", [np.longdouble, np.clongdouble])
+def test_longdouble_with_arrlike(sctype, op):
+    # As of NumPy 2.1, longdouble behaves like other types and can coerce
+    # e.g. lists.  (Not necessarily better, but consistent.)
+    assert_array_equal(op(sctype(3), [1, 2]), op(3, np.array([1, 2])))
+    assert_array_equal(op([1, 2], sctype(3)), op(np.array([1, 2]), 3))
```

Comment:
So yes, the new paths aligns longdouble with others here.  I could guess at this not being right, that would remove the array conversion step.

(It may mean deciding that one test in gh-26905, just can't pass, because we simply cannot deal with subclasses of Python floats even if they get converted to a float array later.  Although, it can go either way, since you can still convert but *only* allow the `was_scalar` path.)